editor/editor.tsx

```diff
@@ -26,6 +26,7 @@ import type { HiGlassSpec } from '@higlass.schema';
 import type { Datum } from '@gosling.schema';
 
 import './editor.css';
+import { RangeMouseEventData } from 'src/core/api';
```

Comment:
Generally we should try to avoid importing from `src/*` from the editor and instead only import what is exposed from `gosling.js` the package. Imports like this will not be available to end users and we can run into the situation where we are relying on behavior that others will not have.

Reply:
Ah, right. I will keep that in mind and avoid importing hidden components.